terraform/app/modules/paas/main.tf

```diff
@@ -35,3 +40,9 @@ resource cloudfoundry_route web_app_route {
   space    = data.cloudfoundry_space.space.id
   hostname = local.web_app_name
 }
+
+resource cloudfoundry_user_provided_service logging {
+  name             = local.logging_service_name
+  space            = data.cloudfoundry_space.space.id
+  syslog_drain_url = var.logstash_url
```

Comment:
Where is this being set?

Reply:
Good question. I need to set that up in terraform.tf

Reply:
And also supply it from a github secret in .github/workflows/deploy.yml

Reply:
Much like RAILS_MASTER_KEY

Reply:
I think with the current setup, you pass it to `secret_paas_app_env_values`, which gets merged into `paas_app_env_values`, which is stored in `app_env_values`, which I _think_ is already a variable that you could use.

So in here, you could have `var.app_env_values. SYSLOG_DRAIN_URL`. 

If you prefer having a variable for it, then when you assign it in another file, I think that's what you should refer to for it to work - then again, deploying PR app is probably the best test for that.

Reply:
This is not an app env var though, so it would seem strange to put it there. This is for terraform to use directly in setting up PaaS